tests/test_project.py

```diff
@@ -314,22 +315,22 @@ class A(FlowProject):
         @A.operation
         @with_job
         def test_context(job):
-            self.assertEqual(os.getcwd(), job.ws)
+            assert os.getcwd() == job.ws
 
         project = self.mock_project()
         with add_cwd_to_environment_pythonpath():
             with switch_to_directory(project.root_directory()):
                 starting_dir = os.getcwd()
                 with redirect_stderr(StringIO()):
                     A().run()
-                self.assertTrue(os.getcwd(), starting_dir)
+                assert os.getcwd(), starting_dir
```

Comment:
Looks to me like this test had originally been incorrect. This should probably be:
```suggestion
                assert os.getcwd() == starting_dir
```